evansmusomi/burger-builder
# Sets Up Redux Dev Tools and Prepares Folder Structure

*What this does*
- Enables redux store introspection through browser dev tools
- Re-organizes store folder structure for future enhancements

*How to verify*
- A Dev can see store and action updates using Redux DevTools when using the app
- A Dev can see the store folder organized into reducer and action folders
